cubical/main.py

```diff
@@ -536,7 +536,7 @@ def prelog_print(level, message):
         chunks_per_tile, tile_list = ms.define_chunk(GD["data"]["time-chunk"], GD["data"]["rebin-time"],
                                             GD["data"]["freq-chunk"],
                                             chunk_by=chunk_by, chunk_by_jump=jump,
-                                            chunks_per_tile=chunks_per_tile, max_chunks_per_tile=max_chunks_per_tile)
+                                            chunks_per_tile=chunks_per_tile, max_chunks_per_tile=GD["dist"]["max-chunks"])
```

Comment:
What is this change? Looks suspicious to me.

Reply:
I have updated it to this to the one on master.